client/features/accountPage/Account.jsx

```diff
@@ -78,17 +70,17 @@ class Account extends React.Component {
           <h2>Subscription Info</h2>
           <ul>Subscriptions: {this.state.platforms.map((sub) => {
             if (sub.isSelected) {
-              return sub.name + ' '
+              return sub.name + ' ';
             }
           }
           )}</ul>
-          <ul>Total Cost: {this.state.platforms.filter(({isSelected}) => isSelected === true).reduce((sum, subs) => {
-              return sum + subs.cost
-              }, 0)}</ul>
+          <ul>Total Cost: {this.state.platforms.filter(({ isSelected }) => isSelected === true).reduce((sum, subs) => {
+            return sum + subs.cost;
+          }, 0)}</ul>
```

Comment:
The function reduce still blows my mind. Well executed.

Reply:
That's Taniel's